src/Sdk/DTPipelines/workflow-v1.0.json

```diff
@@ -71,7 +71,8 @@
           "env": "job-env",
           "outputs": "job-outputs",
           "defaults": "job-defaults",
-          "steps": "steps"
+          "steps": "steps",
+          "snapshot": "snapshot"
```

Comment:
is this schema change coming from the actions-service or the parser?
i am trying to find the source and make sure they match each other.

Reply:
It was added to the C# parser in this PR: https://github.com/github/actions-workflow-parser/pull/406

The Go parser hasn't been updated yet, that's happening in a separate batch: https://github.com/github/hosted-runners/issues/528